hack/compose/docker-compose.yml

```diff
@@ -102,6 +102,16 @@ services:
       MINIO_SECRET_KEY: secret123
     command: server /minio-server/export
 
+  prometheus:
+    network_mode: host
+    image: prom/prometheus
+    depends_on:
+      - omni
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
```

Comment:
file not included in PR?

Reply:
yep and that's not the only file I forgot :) Added both